src/load

```diff
@@ -284,6 +301,31 @@ def fetch_data_feed(appctx, data_name, args):
             data=item
         )
 
+    # process the EPSS data
+    def get_epss_data(item):
+        with gzip.open(item.raw, mode='rt') as csv_file:
+            # csv_reader = csv.DictReader(csv_file) DictReader doens't work since the 1st row in the csv file is not header
+            csv_reader = csv.reader(csv_file)
+
+            # Skip the first row as it just contains the '#model_version:v2023.03.01' and 'score_date:2024-02-12T00:00:00+0000'
+            first_row = next(csv_reader)
+            # score_date also needed to be extracted
+            # Get the second element and split it on ':'
+            date_string = first_row[1].split(':')[1]
+            # Split the date_string on 'T' and take the first part (which is the date)
+            date_value = date_string.split('T')[0]
+
+            # Extract information of each row as a dictionary
+            fieldnames = next(csv_reader)  # the next row is the header information of csv file but we define new names for headers
+            fieldnames_new = ['cve', 'score', 'percentile', 'date']
+            epss_data = []
+            for row in csv_reader:
+                row.append(date_value)
+                epss_data.append(dict(zip(fieldnames_new, row)))
```

Comment:
build the epss_data as a dict with cve-id as keys instead of an list where each value would be a dict with the corresponding epss score values. This will facilitate an easy search for the epss data later on.